k8up/templates/prometheus/prometheusrule.yaml

```
@@ -0,0 +1,41 @@
+{{- if and .Values.metrics.prometheus.enabled .Values.metrics.prometheus.installDefaultRules -}}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ include "k8up.fullname" . }}-rule
+  labels:
+    {{- include "k8up.labels" . | nindent 4 }}
+spec:
+  groups:
+    - name: K8up
+      rules:
+      - alert: baas_last_errors
```

Let's keep naming consistent
```suggestion
      - alert: K8upResticErrors
```